README.md

```diff
@@ -221,8 +221,6 @@ See [https://prejournal.org/example](https://prejournal.org/example) for some ex
 In traditional (GAAP / double entry) bookkeeping, the journal already makes important choices about the system boundaries of an organisation and about depreciation time scales. For instance, if on a given day I bought a laptop and a banana, and then import my bank statement into a generic bookkeeping software package, the first transaction might get booked from `assets : bank : checking` to `assets : equipment : computers` and the other might be journaled as `liabilities : creditcard` to `expenses : groceries`.
 
 Assets, liabilities, and expenses are fundamentally different in traditional bookkeeping, but the act of buying a laptop with your debit card is not fundamentally different from the act of buying a banana with your credit card, and when you federate bookkeeping systems, the local choices about what is an expense (something that lasts less than a month, like a banana) and what is an asset (something that lasts more than a month, like a laptop) should not get exported. That's why we are now experimenting with the federation of bookkeeping systems at the pre-journal phase.
-
-
```

Comment:
Why remove these?

Reply:
These are empty lines.
I didn't remove the https://github.com/pondersource/prejournal/blob/abf570668bacf6182a54b4f4665fb694234bea57/README.md#why